autoblocks/_impl/api/client.py

```diff
@@ -281,17 +281,17 @@ def get_local_test_result(self, test_case_result_id: str) -> AutoblocksTestCaseR
         # Convert evaluations to Evaluation objects
         evaluations = [
             EvaluationWithEvaluatorId(
-                evaluator_id=eval.get("evaluatorId"),
-                score=eval.get("score"),
-                passed=eval.get("passed"),
-                metadata=eval.get("metadata", {}),
-                threshold=eval.get("threshold"),
+                evaluator_id=eval["evaluatorId"],
+                score=eval["score"],
+                passed=eval["passed"],
```

Comment:
this should actually stay as .get since its optional - https://github.com/autoblocksai/autoblocks/blob/main/apps/public-api/src/testing/types.ts#L43